### ISSUE TITLE
508-defect-3 [SCREENREADER]: Issues to review, names of issues not tagged as headings

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#25185]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**VFS Point of Contact:** Angela

<h2>User Story or Problem Statement:</h2>
As a screen reader user, I often navigate by headings when skipping between important sections of the page or looking for important landmarks.
<h2>Details</h2>
On the page where the user can state their reasons for disagreeing with the decision regarding each disability, the names of the individual issues are not tagged as headings. This would be helpful for screen reader users who use heading navigation to skip between important components of a page.
<h2>Acceptance Criteria</h2>
Names of issues are level 3 headings.
<h2>Steps to Recreate</h2>
<ul>
<li>Access

https://staging.va.gov/decision-reviews/board-appeal/request-board-appeal-form-10182/introduction

And sign in with vetsgov+test238@id.me</li>
<li>Proceed to step two and add at least one disability</li>
<li>Proceed to the next screen, which allows you to choose the areas of disagreement with the decision regarding each disability.</li>
</ul>

**Comment by Mottie** [comment]
Fix merged into master. It should be available in staging soon.

**Comment by joshkimux** [comment]
<img width="1077" alt="Screen Shot 2021-07-16 at 11 52 36 AM" src="https://user-images.githubusercontent.com/14154792/125975181-c13f9a3c-b941-4a3f-9454-a694838ad173.png">
Checkbox issue is resolved, but more problems seem to be emerging on the review page specifically around editing 😢

**Comment by Mottie** [comment]
@joshkimux I'm pretty sure [the issue in the screenshot was fixed](https://github.com/department-of-veterans-affairs/vets-website/pull/17975), and [the associated issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/27096#issuecomment-881549828) was also closed. Can we close this issue?

**Comment by joshkimux** [comment]
Yes sir!
